### PR TITLE
Adding instructions for updating T2 firmware until update works

### PR DESCRIPTION
**Pull request by johnnyman727**

### CODE REVIEW

**Comment by johnnyman727** [comment]
Note that the firmware updating instructions won't exist until https://github.com/tessel/t2-firmware/pull/51 is merged.

@rwaldron want to try this out?

File: README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ Join the [conversation on Slack](https://tessel-slack.herokuapp.com/), our proje
...
...ssues/81) but it hasn't been built yet. For now, go ahead and update your co-processor firmware by following the instructions [here](https://github.com/tessel/t2-firmware/#flashing). Then update the OpenWRT image on the primary processor by either [building it yourself](https://github.com/tessel/openwrt-tessel) or downloading [a prebuilt binary](https://tessel.slack.com/archives/engineering/p1435040051000036), transferring it to your T2 (often accomplished with `scp -i ~/.tessel/id_rsa PATH_TO_IMAGE root@IP_ADDR_OF_T2:/tmp`) and running `sysupgrade /tmp/PATH_TO_IMAGE` on your T2. Make sure it's uploaded to the `/tmp` folder on T2 - any other folder will ruin the upgrade. If you need to find the IP Address of your T2 but don't know how, first connect it to your network (`t2 wifi -n YOUR_SSID -p YOUR_PASSWORD`), then run `t2 list` to make sure it's actually connected over the LAN, and note its name. Then run `ping YOUR_NAME.local` in order to see the IP address (you can also give it a simpler name first with `t2 rename NEW_NAME`. If you already know what dterm is, you can just dterm into the device and run `ifconfig`. Wait for it to automatically restart itself and you're good to go!
```
Comment (rwaldron) [inline README.md]: The `#flashing` url fragment isn't present here: https://github.com/tessel/t2-firmware/#flashing

Comment (johnnyman727) [inline README.md]: Yeah, because this PR is dependent on tessel/t2-firmware#51

Comment (rwaldron) [inline README.md]: `https://tessel.slack.com/archives/engineering/p1435040051000036` => `https://kevinmehall.net/tmp/openwrt/openwrt-ramips-mt7620-tessel-squashfs-sysupgrade.bin`

Comment (rwaldron) [inline README.md]: Got it!

**Comment by johnnyman727** [comment]
@rwaldron it looks like when #176 was merged, your unit tests didn't get run through Grunt.

**Pull request merged**